src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -77,16 +78,21 @@ export const styles = StyleSheet.create({
     lineHeight: 19.36,
     marginBottom: 4
   },
-  deletedText: {
+  deletedMyText: {
     color: colors.light_silver,
     fontSize: normalizeFontSize(15),
     fontStyle: 'italic'
   },
+  deletedTargetText: {
+    color: colors.blackgrey,
+    fontSize: normalizeFontSize(15),
+    fontStyle: 'italic'
+  },
   avatar: {
     width: dimen.normalizeDimen(24),
     height: dimen.normalizeDimen(24),
     borderRadius: 15,
-    marginLeft: AVATAR_LEFT_MARGIN
+    marginLeft: dimen.normalizeDimen(AVATAR_LEFT_MARGIN)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to be for distinguishing between deleted messages from the user and the target. This is a good approach as it allows for different styles for each case. However, it's important to ensure that the new constants `deletedMyText` and `deletedTargetText` are used correctly in the rest of the codebase.
```diff
-  deletedText: {
+  deletedMyText: {
     color: colors.light_silver,
     fontSize: normalizeFontSize(15),
     fontStyle: 'italic'
   },
+  deletedTargetText: {
+    color: colors.blackgrey,
+    fontSize: normalizeFontSize(15),
+    fontStyle: 'italic'
+  },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->